### PR TITLE
<TextArea>

### PR DESCRIPTION
**Pull request by ryotogashi**

### CODE REVIEW

**Comment by ryotogashi** [comment]
@axross I separate TextArea from TextInput. Can you check?

File: components/TextArea/TextArea.tsx [hunk 1]
```diff
@@ -0,0 +1,75 @@
+import * as React from "react";
+import styled, { FlattenSimpleInterpolation, css } from "styled-components";
+
+interface Props extends React.Attributes {
+  fullWidth?: boolean;
+  placeholder?: string;
+  rows?: number;
+  size?: InputSize;
+  onChange?: React.ChangeEventHandler<HTMLInputElement | HTMLTextAreaElement>;
```
Comment (axross) [inline components/TextArea/TextArea.tsx]: :+1:

Comment (axross) [reply components/TextArea/TextArea.tsx]: However, it cannot be `HTMLInputElement`, right?

File: components/TextArea/TextArea.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+export default {
+  title: "TextArea",
+  component: TextArea
+};
+
+export const example = () => (
+  <TextArea
+    size={select("size", SIZES, undefined)}
+    onChange={action("inputed")}
+  />
```
Comment (axross) [inline components/TextArea/TextArea.stories.tsx]: It would be better if `placeholder`, `rows` and `fullWidth` are customizable with knobs.

**Comment by axross** [comment]
@ryotogashi Is this okay? (note that it's Full Width ones)

Screen Shot 2020-03-17 at 7 18 22 PM

**Comment by axross** [comment]
@ryotogashi Seems some `font-family` is wrongly set 🤔

**Comment by axross** [comment]
As mentioned on LINE, I'll take it.

**Pull request closed without merging**